contracts/LifToken.sol

```diff
@@ -457,14 +469,20 @@ contract LifToken is Ownable, ERC20, SafeMath, PullPayment {
         giveVotes(msg.sender, to);
       }
 
-      TransferData(msg.sender, to, value, data);
+      if (doCall && to.call(data))
+        TransferData(msg.sender, to, value, data);
+      else if (!doCall)
```

Comment:
So when `!(doCall && to.call(data))` it seems like we just return true, is that correct? Like it fails silently, or do we just expect `to.call(data)` to throw when it fails?